pkg/commands/git.go

```diff
@@ -15,13 +15,34 @@ import (
 	gogit "gopkg.in/src-d/go-git.v4"
 )
 
+// ErrGitRepositoryInvalid is emitted when we run a git command in a folder
+// to check if we have a valid git repository and we get an error instead
+var ErrGitRepositoryInvalid = fmt.Errorf("can't find a valid git repository in current directory")
```

Comment:
Is there a particular advantage of fmt.Errorf() over errors.New()? If possible I'd like to only have one appear in the codebase

Reply:
No it's a mistake as well you don't any purpose here for fmt and it's for this kind of errror you should use the error package, I will fix it.